src/components/learner-credit-management/data/hooks/useEnterpriseGroup.js

```diff
@@ -0,0 +1,28 @@
+import { useQuery } from '@tanstack/react-query';
+import { camelCaseObject } from '@edx/frontend-platform/utils';
+
+import { learnerCreditManagementQueryKeys } from '../constants';
+import LmsApiService from '../../../../data/services/LmsApiService';
+
+/**
+ * Retrieves a enterprise group by UUID from the API.
+ *
+ * @param {*} queryKey The queryKey from the associated `useQuery` call.
+ * @returns The enterprise group object
+ */
+const getEnterpriseGroup = async ({ subsidyAccessPolicy }) => {
+  if (!subsidyAccessPolicy.groupAssociations || subsidyAccessPolicy.groupAssociations.length === 0) {
```

Comment:
nit: we're already using [_.isEmpty](https://lodash.com/docs/4.17.15#isEmpty) [here](https://github.com/openedx/frontend-app-admin-portal/blob/1ea92336d03006fc82f8aa3f1d464b8e3094e97b/src/components/settings/SettingsLMSTab/utils.js#L1).
```suggestion
  if (_.isEmpty(subsidyAccessPolicy.groupAssociations)) {
```